Software/app/basic_azurertos/conf/app_conf.h

```diff
@@ -0,0 +1,42 @@
+/** Copyright © 2021 The Things Industries B.V.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+/**
+ * @file app_conf.h
+ * @brief Common configuration file for GNSE applications
+ *
+ * @copyright Copyright (c) 2021 The Things Industries B.V.
+ *
+ */
+
+#ifndef APP_CONF_H
+#define APP_CONF_H
+
+#define GNSE_TINY_TRACER_ENABLE 1
+
+#define APP_DEMO_STACK_SIZE 1024
+#define APP_DEMO_BYTE_POOL_SIZE 9120
+#define APP_DEMO_QUEUE_SIZE 100
+
+/* The rate at which the Tx task sends to the queue. */
+#define APP_TX_DELAY                 (5000)
+
+/* The length of time the LED will remain on for.  It is on just long enough
+to be able to see with the human eye so as not to distort the power readings too
+much. */
+#define APP_LED_TOGGLE_DELAY         (20)
```

Comment:
Extremely nitpicky, but basic_freertos has essentially the same config variable but only it's called LED_TOGGLE_DELAY, so maybe for consistency we could change this or the other.

Reply:
Fixed this but in the future all app conf will have an APP prefix